### PR TITLE
adding post about effective online meetings

### PR DESCRIPTION
**Pull request by weaverbel**

### CODE REVIEW

**Comment by weaverbel** [comment]
I created this with a push from Karin Lagesen

File: _posts/2017/11/2017-11-08-online-meetings.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+4. Move to a quiet place if you can so outside noise does not intrude on others.
+
+5. Please don't be the person who comes in late, without a headset, and makes a lot of noise from screechy feedback!
+
+#### In the meeting:
+
+The Chair sets the tone. The Chair introduces him or herself, explains how the meeting will be conducted, and what expectations there are of attendees. The Chair then calls on any notetaker for the meeting to introduce him or herself.
+
+The chair greets each person as that person joins the meeting. Once themeeting starts, the Chair calls on each person in turn to say their name, where they are from, and why they are attending, e.g.instructor checkout, workshop debrief.
```
Comment (tracykteal) [inline _posts/2017/11/2017-11-08-online-meetings.md]: the meeting

**Comment by weaverbel** [comment]
Thanks for the typo catch @tracykteal ! Waiting for extra input from @karinlag so please don't merge this yet.

**Comment by tracykteal** [comment]
I'd also add that it's good to designate a notetaker for an online meeting.

File: _posts/2017/11/2017-11-08-online-meetings.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+4. Move to a quiet place if you can so outside noise does not intrude on others.
+
+5. Please don't be the person who comes in late, without a headset, and makes a lot of noise from screechy feedback!
+
+#### In the meeting:
+
+The Chair sets the tone. The Chair introduces him or herself, explains how the meeting will be conducted, and what expectations there are of attendees. The Chair then calls on any notetaker for the meeting to introduce him or herself.
+
+The chair greets each person as that person joins the meeting. Once the meeting starts, the Chair calls on each person in turn to say their name, where they are from, and why they are attending, e.g.instructor checkout, workshop debrief.
+
```
Comment (karinlag) [inline _posts/2017/11/2017-11-08-online-meetings.md]: This last bit becomes a bit specific for the Carpentries, maybe generalize a bit? Wouldn't hurt to make this a universal resource :)

File: _posts/2017/11/2017-11-08-online-meetings.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+3. Mute your cell phone in case it rings or pings during the meeting.
+
+4. Move to a quiet place if you can so outside noise does not intrude on others.
+
+5. Please don't be the person who comes in late, without a headset, and makes a lot of noise from screechy feedback!
+
+#### In the meeting:
+
+The Chair sets the tone. The Chair introduces him or herself, explains how the meeting will be conducted, and what expectations there are of attendees. The Chair then calls on any notetaker for the meeting to introduce him or herself.
+
```
Comment (karinlag) [inline _posts/2017/11/2017-11-08-online-meetings.md]: We highly recommend that there is a designated notetaker. After the introductory round, we recommend that the chair asks the notetaker to introduce themselves, and to explain how and where notes will be stored and distributed.

File: _posts/2017/11/2017-11-08-online-meetings.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+The chair greets each person as that person joins the meeting. Once the meeting starts, the Chair calls on each person in turn to say their name, where they are from, and why they are attending, e.g.instructor checkout, workshop debrief.
+
+The Chair asks people who wish to speak to write the word `hand` in the chat window. The Chair then calls on speakers in turn. 
+
+Notes are recorded on an etherpad. Any notes posted to the chat window during the meeting should be transferred to the etherpad so they are not lost when the zoom is closed.
+
```
Comment (karinlag) [inline _posts/2017/11/2017-11-08-online-meetings.md]: Notes are recorded on an etherpad or a similar shared medium. Any notes posted to chats during the meeting should be transferred to the shared medium so they are not lost when the video conference is closed.

File: _posts/2017/11/2017-11-08-online-meetings.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+---
+ 
+Online meetings are a fact of life for most of us in the Carpentries. So how do we make them as effective as we can?
+
+This post came about after the Twitter discussion generated by this [tweet from Titus Brown](https://twitter.com/ctitusbrown/status/926165114948001792).
+
+Here are some points I have thought of. Feel free to comment.
+
+#### Before the meeting:
+
```
Comment (karinlag) [inline _posts/2017/11/2017-11-08-online-meetings.md]: Familiarize yourself with the agenda for the meeting. Are there things you need to figure out beforehand? Are there things you are wondering about? If there is no agenda, ask the Chair to send one out.

File: _posts/2017/11/2017-11-08-online-meetings.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+The chair greets each person as that person joins the meeting. Once the meeting starts, the Chair calls on each person in turn to say their name, where they are from, and why they are attending, e.g.instructor checkout, workshop debrief.
+
+The Chair asks people who wish to speak to write the word `hand` in the chat window. The Chair then calls on speakers in turn. 
+
+Notes are recorded on an etherpad. Any notes posted to the chat window during the meeting should be transferred to the etherpad so they are not lost when the zoom is closed.
+
+#### Three rules to make for better meetings
+
+- Be on time - it is very disruptive when people join an online meeting after it has started.
```
Comment (karinlag) [inline _posts/2017/11/2017-11-08-online-meetings.md]: Be prepared for the meeting - meetings where people know what's going on are more effective and will take less time.

**Comment by karinlag** [comment]
Added my couple of cents, hope they make sense!

**Comment by weaverbel** [comment]
Thanks @tracykteal and @karinlag I have incorporated your suggestions and think we are ready for take off.

**Pull request merged**